pontoon/base/formats/lang.py

```diff
@@ -147,6 +149,10 @@ def visit_entity(self, node, (string, translation)):
             tags = [m.group(1) for m in tag_matches]
             translation = translation[:tag_matches[0].start()].strip()
 
+        if translation == '':
+            raise Exception('Blank translation for key {key} is not allowed in '
+                            'langfiles.'.format(key=string))
+
```

Comment:
Nit: FHR fails with en-US already, so with original string, not translation. You might generalize this and use string instead of translation.